src/main/java/com/emc/mongoose/storage/driver/pravega/kvs/RoutingKeyFunction.java

```diff
@@ -0,0 +1,10 @@
+package com.emc.mongoose.storage.driver.pravega.kvs;
+
+import com.emc.mongoose.base.item.DataItem;
+import java.util.function.Function;
+
+public interface RoutingKeyFunction<I extends DataItem>
```

Comment:
Do we need to use Hashing Key instead of Routing Key? This can be confusing.

Reply:
I left it mainly to remember that we may be able to reuse that code when we het to key families. I'm not sure we can use these classes directly, so I'd prefer they stay rountingKey until we are sure they are working for KVT

Reply:
The name should change, that's without doubt